env.go.jp,"HTTPSConnectionPool(host='env.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0a88c5f760>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='env.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0a87acc160>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
epm.com.co,"HTTPSConnectionPool(host='epm.com.co', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0a857233a0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='epm.com.co', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0a87a979a0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
eprice.it,"HTTPSConnectionPool(host='www.eprice.it', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.eprice.it', port=443): Read timed out. (read timeout=2)"
equipement.gov.ma,"HTTPSConnectionPool(host='equipement.gov.ma', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0a852f8670>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='equipement.gov.ma', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0a85723790>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
erasmusmc.nl,"HTTPSConnectionPool(host='erasmusmc.nl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='erasmusmc.nl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
erbakan.edu.tr,"HTTPSConnectionPool(host='erbakan.edu.tr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='erbakan.edu.tr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
erciyes.edu.tr,"HTTPSConnectionPool(host='erciyes.edu.tr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='erciyes.edu.tr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
erdieindustries.co,"HTTPSConnectionPool(host='erdieindustries.co', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='erdieindustries.co', port=80): Read timed out. (read timeout=2)"
eroski.es,"HTTPSConnectionPool(host='eroski.es', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='eroski.es', port=443): Read timed out. (read timeout=2)"
esaunggul.ac.id,"HTTPSConnectionPool(host='esaunggul.ac.id', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='esaunggul.ac.id', port=443): Read timed out. (read timeout=2)"
escape.com.au,"HTTPSConnectionPool(host='www.escape.com.au', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.escape.com.au', port=443): Read timed out. (read timeout=2)"
escuelasoulsurf.cl,"HTTPSConnectionPool(host='escuelasoulsurf.cl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0a87ad4df0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='escuelasoulsurf.cl', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0a83e0b790>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
eskom.co.za,"HTTPSConnectionPool(host='eskom.co.za', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0a86e5fbe0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='eskom.co.za', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0a83b6f760>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
esl-lounge.com,"HTTPSConnectionPool(host='esl-lounge.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='esl-lounge.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
eso.lt,"HTTPSConnectionPool(host='eso.lt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='eso.lt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
esocietyplatform.nl,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
espe.edu.ec,"HTTPSConnectionPool(host='espe.edu.ec', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0a83a69fd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='espe.edu.ec', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0a83a157c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
espoch.edu.ec,"HTTPSConnectionPool(host='espoch.edu.ec', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='espoch.edu.ec', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
esteri.it,"HTTPSConnectionPool(host='esteri.it', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0a876dadf0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='esteri.it', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0a82856bb0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
estravel.lt,"HTTPSConnectionPool(host='www.estravel.lt', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.estravel.lt', port=443): Read timed out. (read timeout=2)"
etimad.sa,"HTTPSConnectionPool(host='etimad.sa', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='etimad.sa', port=443): Read timed out. (read timeout=2)"
etmall.com.tw,"HTTPSConnectionPool(host='etmall.com.tw', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'etmall.com.tw' doesn't match 'www.etmall.com.tw'"")))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
etno-muzej.si,"HTTPSConnectionPool(host='etno-muzej.si', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0a81020ca0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='etno-muzej.si', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0a87a977c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
euro-nunta.ro,"HTTPConnectionPool(host='www.euro-nunta.ro', port=80): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.euro-nunta.ro', port=80): Read timed out. (read timeout=2)"
euro.com.pl,"HTTPSConnectionPool(host='www.euro.com.pl', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.euro.com.pl', port=443): Read timed out. (read timeout=2)"
eurocartv.nl,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
eurolines.fr,"HTTPSConnectionPool(host='eurolines.fr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0a7faaffd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='eurolines.fr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0a81e7bd60>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
evidus.com,"HTTPSConnectionPool(host='evidus.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0a7e129400>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='evidus.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0a7f82c310>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
evkur.com.tr,"HTTPSConnectionPool(host='evkur.com.tr', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f0a7e1bab20>, 'Connection to evkur.com.tr timed out. (connect timeout=2)'))","HTTPConnectionPool(host='evkur.com.tr', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f0a86ec1850>, 'Connection to evkur.com.tr timed out. (connect timeout=2)'))"
evz.ro,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
excelsior.com.mx,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
expatchoice.asia,"HTTPSConnectionPool(host='www.expatchoice.asia', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.expatchoice.asia', port=443): Read timed out. (read timeout=2)"
extra.com.br,"HTTPConnectionPool(host='www.extra.com.br', port=80): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.extra.com.br', port=80): Read timed out. (read timeout=2)"
extradom.pl,"HTTPSConnectionPool(host='extradom.pl', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='extradom.pl', port=80): Read timed out. (read timeout=2)"
extrafarma.com.br,"HTTPSConnectionPool(host='extrafarma.com.br', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0a87c0d940>: Failed to establish a new connection: [Errno 113] No route to host'))","HTTPConnectionPool(host='extrafarma.com.br', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0a83b103d0>: Failed to establish a new connection: [Errno 113] No route to host'))"
extrasports.com,Exceeded 30 redirects.,"HTTPSConnectionPool(host='www.extrasports.com', port=443): Read timed out. (read timeout=2)"
fajar.co.id,"HTTPSConnectionPool(host='fajar.co.id', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='fajar.co.id', port=443): Read timed out. (read timeout=2)"
famigliacristiana.it,"HTTPSConnectionPool(host='famigliacristiana.it', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0a8704e4f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='famigliacristiana.it', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0a7e6b1910>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
family.co.jp,"HTTPSConnectionPool(host='family.co.jp', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f0a7c44ec40>, 'Connection to family.co.jp timed out. (connect timeout=2)'))","HTTPConnectionPool(host='family.co.jp', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f0a80d6f3d0>, 'Connection to family.co.jp timed out. (connect timeout=2)'))"
famitsu.com,"HTTPSConnectionPool(host='famitsu.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='famitsu.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
fans.lv,"HTTPSConnectionPool(host='www.fans.lv', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.fans.lv', port=443): Read timed out. (read timeout=2)"
farbotex.it,"HTTPSConnectionPool(host='farbotex.it', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'farbotex.it' doesn't match either of '*.aruba.it', 'aruba.it'"")))","HTTPConnectionPool(host='www.farbotex.it', port=80): Read timed out. (read timeout=2)"
farfetch.com,"HTTPSConnectionPool(host='www.farfetch.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.farfetch.com', port=443): Read timed out. (read timeout=2)"
farmaciasguadalajara.com,"HTTPSConnectionPool(host='farmaciasguadalajara.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0a87869fa0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='farmaciasguadalajara.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0a871e4b20>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
farmaciasportuguesas.pt,"HTTPSConnectionPool(host='farmaciasportuguesas.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0a7e0f3460>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='farmaciasportuguesas.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0a7fa8c040>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
farmaku.com,"HTTPSConnectionPool(host='www.farmaku.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.farmaku.com', port=443): Read timed out. (read timeout=2)"
farmhopping.com,"HTTPSConnectionPool(host='farmhopping.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='farmhopping.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
fashion-press.net,"HTTPSConnectionPool(host='fashion-press.net', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f0a87874cd0>, 'Connection to fashion-press.net timed out. (connect timeout=2)'))","HTTPConnectionPool(host='fashion-press.net', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f0a828fc280>, 'Connection to fashion-press.net timed out. (connect timeout=2)'))"
fashionismo.com.br,"HTTPSConnectionPool(host='fashionismo.com.br', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0a862e3160>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPSConnectionPool(host='fashionismo.com.br', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0a7ea0a130>: Failed to establish a new connection: [Errno 111] Connection refused'))"
fassil.com.bo,"HTTPSConnectionPool(host='fassil.com.bo', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.fassil.com.bo', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
fazenda.gov.br,"HTTPSConnectionPool(host='fazenda.gov.br', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0a86aa3910>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='fazenda.gov.br', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0a7e44b3a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
fbr.gov.pk,"HTTPSConnectionPool(host='fbr.gov.pk', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f0a8781d610>, 'Connection to fbr.gov.pk timed out. (connect timeout=2)'))","HTTPSConnectionPool(host='fbr.gov.pk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
fda.gov,"HTTPSConnectionPool(host='fda.gov', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0a86d86b80>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='fda.gov', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0a7e354550>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
fdl.com.bd,"HTTPSConnectionPool(host='fdl.com.bd', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f0a7e348b50>, 'Connection to fdl.com.bd timed out. (connect timeout=2)'))","HTTPConnectionPool(host='fdl.com.bd', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f0a858287c0>, 'Connection to fdl.com.bd timed out. (connect timeout=2)'))"
felix.si,"HTTPSConnectionPool(host='felix.si', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='felix.si', port=80): Read timed out. (read timeout=2)"
femina.co.id,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
feol.hu,"HTTPSConnectionPool(host='www.feol.hu', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.feol.hu', port=443): Read timed out. (read timeout=2)"
fero-term.hr,"HTTPSConnectionPool(host='fero-term.hr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.fero-term.hr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
ffos.hr,"HTTPSConnectionPool(host='ffos.hr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0a7ea15250>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ffos.hr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0a8297b400>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
fhuce.edu.uy,"HTTPSConnectionPool(host='fhuce.edu.uy', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0a7c3c8a90>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='fhuce.edu.uy', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0a872494f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ficohsa.com,"HTTPSConnectionPool(host='ficohsa.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f0a883cff40>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ficohsa.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0a86f671f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
filmitv.rs,"HTTPSConnectionPool(host='filmitv.rs', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='filmitv.rs', port=443): Read timed out. (read timeout=2)"
filmmodu.org,"HTTPSConnectionPool(host='filmmodu.org', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'filmmodu.org' doesn't match 'www.filmmodu.org'"")))","HTTPSConnectionPool(host='filmmodu.org', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'filmmodu.org' doesn't match 'www.filmmodu.org'"")))"
filmovamista.cz,"HTTPSConnectionPool(host='filmovamista.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))","HTTPSConnectionPool(host='filmovamista.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))"
filmsvina.sk,"HTTPSConnectionPool(host='filmsvina.sk', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='filmsvina.sk', port=80): Read timed out. (read timeout=2)"
fimas.cl,"HTTPSConnectionPool(host='fimas.cl', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='fimas.cl', port=443): Read timed out. (read timeout=2)"
finances.gov.ma,"HTTPSConnectionPool(host='finances.gov.ma', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f0a7dc2d3d0>, 'Connection to finances.gov.ma timed out. (connect timeout=2)'))","HTTPConnectionPool(host='finances.gov.ma', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f0a85f1a250>, 'Connection to finances.gov.ma timed out. (connect timeout=2)'))"
financialmirror.com,"HTTPSConnectionPool(host='www.financialmirror.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.financialmirror.com', port=443): Read timed out. (read timeout=2)"
